redis/server.cc

```diff
@@ -41,7 +41,7 @@ namespace redis_transport {
 
 static logging::logger logging("redis_server");
 
-redis_server::redis_server(distributed<service::storage_proxy>& proxy, distributed<redis::query_processor>& qp, auth::service& auth_service, redis_server_config config)
+redis_server::redis_server(seastar::sharded<service::storage_proxy>& proxy, seastar::sharded<redis::query_processor>& qp, auth::service& auth_service, redis_server_config config)
```

Comment:
I don't mind this change too much, but why did you have to write seastar::sharded and not just sharded?

Reply:
I was not aware there's a preference. I just switched to the fully qualified name in the header and source file for consistency.